### DISCUSSION TITLE
Welcome to pi-zero-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/pi-zero-pytorch#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bombardelli** [comment]
Hi Phil, this group from China opened source their code regarding a robotic diffusion transformer. To load the pre trainee weights they were able to load the pre trained weights of SigLIP and T5 and have their code available, hope that helps with the pi0 code development, thank you for your work 
https://arxiv.org/abs/2410.07864

**Reply by lucidrains** [reply]
yea thanks, i'll prob stick with pali gemma (in particular v2 that just got open sourced)
